data.sql

```diff
@@ -88,4 +88,37 @@ UPDATE
 SET
   owner_id = 5
 WHERE
-  name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+  name IN ('Angemon', 'Boarmon');
+
+INSERT INTO
+  vets (name, age, date_of_graduation)
+VALUES
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-07-08');
+
+INSERT INTO
+  visits (animal_id, vet_id, date)
+VALUES
+  ('1', '1', '2020-5-24'),
+  ('1', '3', '2020-7-22'),
+  ('2', '4', '2021-2-2'),
+  ('3', '2', '2020-1-5'),
+  ('3', '2', '2020-3-8'),
+  ('3', '2', '2020-5-14'),
+  ('4', '3', '2021-5-4'),
+  ('5', '4', '2021-2-24'),
+  ('6', '2', '2019-12-21'),
+  ('6', '1', '2020-8-10'),
+  ('6', '2', '2021-4-7'),
+  ('7', '3', '2019-9-29'),
+  ('8', '4', '2020-10-3'),
+  ('8', '4', '2020-11-4'),
+  ('9', '2', '2019-1-24'),
+  ('9', '2', '2019-5-15'),
+  ('9', '2', '2020-2-27'),
+  ('9', '2', '2020-8-3'),
+  ('10', '3', '2020-5-24'),
+  ('10', '1', '2021-1-11');
+
```

Comment:
- [ ] These particular requirements are missing in your data.sql file. Kindly add them and update your queries

![image](https://github.com/yosaddis/Vet-clinic-database/assets/63473584/3516af5b-1199-4d1f-bc18-5570f62b190b)